src/components/Home.js

```diff
@@ -49,7 +49,7 @@ class Home extends Component {
           : (
             <div className="mint-or-redeem">
               <a href="/deposit" onClick={this.handleClickDeposit}>
-                <button className="blue">
+                <button className="black">
```

Comment:
Can we make this `primary` or similar instead of a color? Or, if we're not using this to distinguish between primary/secondary, we can just target the button as `.mint-or-redeem a` or whatever. Also, obviously this predates our dance here, but the `button` element needs to go, given that it has no functional relevance. We can just style the `a` element appropriately.

Again, not blocking, just something to look at in a follow-up.

Reply:
`.primary` is a good idea

Reply:
Just got rid of all the <button="black"> since they were all instances in which they were wrapping an `<a>` element. Addressed in d820efb